tests/bluechi_test/bluechictl.py

```diff
@@ -33,14 +33,22 @@ def _run(self, log_txt: str, cmd: str, check_result: bool, expected_result: int)
             raise Exception(f"{log_txt} failed with {result} (expected {expected_result}): {output}")
         return result, output
 
-    def list_units(self, node_name: str, check_result: bool = True, expected_result: int = 0) \
+    def list_units(self, node_name: str = None, check_result: bool = True, expected_result: int = 0) \
             -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
-        return self._run(
-            f"Listing units on node {node_name}",
-            f"list-units {node_name}",
-            check_result,
-            expected_result
-        )
+        if node_name:
+            return self._run(
+                f"Listing units on node {node_name}",
+                f"list-units {node_name}",
+                check_result,
+                expected_result
+            )
+        else:
+            return self._run(
+                "Listing units on all nodes",
+                "list-units",
+                check_result,
+                expected_result
+            )
```

Comment:
I'd remove the `else` here:
```suggestion
        return self._run(
            "Listing units on all nodes",
            "list-units",
            check_result,
            expected_result
        )
```

Reply:
Done